src/modules/dashboard/components/CreatePaymentDialog/CreatePaymentDialogForm.css

```diff
@@ -0,0 +1,60 @@
+.tokenAmount {
+  display: grid;
+  width: 100%;
+  grid-template-columns: auto 100px;
```

Comment:
I don't care _that_ much about this, but this isn't a very good use of `grid`

This could have very easily be achieved just with `flex`